server/shovel-config.ts

```diff
@@ -153,16 +153,17 @@ async function integrationFor(
 if (process.env.OUTPUT) {
   ;(async function main() {
     let integrations = await Promise.all([
-      integrationFor('Problems', 'Transfer', [
+      integrationFor('Speedruns', 'Transfer', [
         ['block_num DESC', 'tx_idx DESC', 'log_idx DESC']
       ]),
-      integrationFor('Bodies', 'Transfer', [
-        ['block_num DESC', 'tx_idx DESC', 'log_idx DESC']
+      integrationFor('AnybodyProblem', 'RunCreated'),
+      integrationFor('AnybodyProblem', 'RunSolved', [
+        ['accumulative_time ASC']
       ]),
-      integrationFor('Solver', 'Solved'),
-      integrationFor('Bodies', 'bodyBorn'),
-      integrationFor('Problems', 'bodyAdded'),
-      integrationFor('Problems', 'bodyRemoved')
+      integrationFor('AnybodyProblem', 'LevelCreated'),
+      integrationFor('AnybodyProblem', 'LevelSolved', [['time ASC']]),
+      integrationFor('AnybodyProblem', 'OwnershipTransferred'),
```

Comment:
shouldn't need to listen for this one, it's just about who has admin rights on the contract for configuration